doc/news/_preparation_next_release.md

```diff
@@ -543,6 +543,7 @@ mounted, use `kdb gen -F <plugin>:<file> elektra <parentKey> <outputName>` to lo
 - Added a section on `elektraPluginGetGlobalKeySet` in the plugin tutorial. _(Vid Leskovar)_
 - Added a step-by-step [tutorial](../tutorials/run_all_tests_with_docker.md) for running all tests with Docker. _(Oleksandr Shabelnyk)_
 - Added a step-by-step [tutorial](../tutorials/run_reformatting_script_with_docker.md) for running reformatting scripts with Docker. _(Oleksandr Shabelnyk)_
+- Covered Resolving Missing \*.so Library Error in [tutorial](../tutorials/contributing-clion.md) and [tutorial](../COMPILE.md). _(Oleksandr Shabelnyk)_
```

Comment:
Outdated information (COMPILE.md is not modified).

Reply:
Thank you. I forgot to remove it.